### PR TITLE
feat: initial smoke test

### PR DESCRIPTION
**Pull request by MilanVojnovic95**
- added smoke test for carrot home page
- checking text 
- checking redirections

Note: currently will work on local since I have added additional selector

It can be optimizes and improved but I wanted to have some checks for start and improve it later

### CODE REVIEW

File: packages/frontend/e2e/tests/connect-wallet.test.ts
Comment (guerrap) [inline packages/frontend/e2e/tests/connect-wallet.test.ts:1]: Why is this all commented?

File: packages/frontend/e2e/utils/data/data.ts
Comment (guerrap) [inline packages/frontend/e2e/utils/data/data.ts:1]: Do we need other files under `/data` other than `data.txs`? If that's not the case I'd remove the folder and just have the `data.ts` file.

File: packages/frontend/e2e/tests/smoke.test.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
+import { test } from "../utils/fixtures";
+import { networks, footerLinks } from "../utils/data/data";
+/**
```
Comment (guerrap) [inline packages/frontend/e2e/tests/smoke.test.ts:3]: Let's leave a line between the imports and the rest.

File: packages/frontend/e2e/utils/fixtures.ts [hunk 2]
```diff
@@ -3,10 +3,18 @@ import { initialSetup } from "@synthetixio/synpress/commands/metamask";
 import { setExpectInstance } from "@synthetixio/synpress/commands/playwright";
 import { resetState } from "@synthetixio/synpress/commands/synpress";
 import { prepareMetamask } from "@synthetixio/synpress/helpers";
-
+import { HomePage } from "../utils/pages/homePage";
+/**
```
Comment (guerrap) [inline packages/frontend/e2e/utils/fixtures.ts:7]: Same here, new line.

File: packages/frontend/e2e/utils/pages/homePage.ts [hunk 3]
```diff
@@ -0,0 +1,294 @@
+import { BasePage } from "../pages/basePage";
+import {
+    campaignData,
+    urls,
+    networks,
+    templateData,
+    textData,
+    wallets,
+} from "../data/data";
+/**
```
Comment (guerrap) [inline packages/frontend/e2e/utils/pages/homePage.ts:10]: Here too.

File: packages/frontend/package.json [hunk 4]
```diff
@@ -35,12 +35,14 @@
         "config-react-env": "./scripts/config-react-env.js",
         "start:staging": "STAGING=true craco start",
         "start": "craco start",
-        "test:e2e": "yarn prepare-fathom && playwright test"
+        "test:e2e": "yarn prepare-fathom && playwright test",
+        "smoke": "yarn playwright test"
```
Comment (guerrap) [inline packages/frontend/package.json:39]: Why a new `smoke` script if we run the same `playwright test` like the script above?

File: packages/frontend/e2e/tests/connect-wallet.test.ts
Comment (MilanVojnovic95) [reply packages/frontend/e2e/tests/connect-wallet.test.ts:1]: I forgot to uncomment it

File: packages/frontend/e2e/utils/data/data.ts
Comment (MilanVojnovic95) [reply packages/frontend/e2e/utils/data/data.ts:1]: I will leave only the `data.ts` file for now

File: packages/frontend/package.json (hunk 4 above)
Comment (MilanVojnovic95) [reply packages/frontend/package.json:39]: yes I wanted to have separate script just for smoke test and the other one for all tests I'will just edit the existing one to be ` "test:e2e": "yarn prepare-fathom && yarn playwright test"` in order to work

File: packages/frontend/package.json [hunk 5]
```diff
@@ -35,8 +35,7 @@
         "config-react-env": "./scripts/config-react-env.js",
         "start:staging": "STAGING=true craco start",
         "start": "craco start",
-        "test:e2e": "yarn prepare-fathom && playwright test",
```
Comment (guerrap) [inline packages/frontend/package.json]: We can omit the `yarn` before `playwright test`. so just have:

`yarn prepare-fathom && playwright test`

Comment (MilanVojnovic95) [reply packages/frontend/package.json]: I will edit it now but I am getting error `command not found: playwright test` if I ommit yarn

Comment (guerrap) [reply packages/frontend/package.json]: Running it on my machine it works fine without the `yarn`, anyway let's leave it if it causes issues.

**Review by guerrap — APPROVED** [review]

**Pull request merged**